x/datapool/client/cli/txPool.go

```diff
@@ -35,3 +42,75 @@ func CmdRegisterDataValidator() *cobra.Command {
 	flags.AddTxFlagsToCmd(cmd)
 	return cmd
 }
+
+func CmdCreatePool() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create-pool [pool params file]",
+		Short: "create a new data pool",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			msg, err := NewCreatePoolMsg(clientCtx, args[0])
+			if err != nil {
+				return err
+			}
+
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	flags.AddTxFlagsToCmd(cmd)
+	return cmd
+}
+
+func NewCreatePoolMsg(clientCtx client.Context, file string) (sdk.Msg, error) {
+	var poolParamsInput createPoolInput
+
+	contents, err := ioutil.ReadFile(file)
+	if err != nil {
+		return nil, fmt.Errorf("failed to read file")
+	}
+
+	dec := json.NewDecoder(bytes.NewReader(contents))
+
+	if err := dec.Decode(&poolParamsInput); err != nil {
+		return nil, err
+	}
```

Comment:
How about modifying it like this?
```suggestion
	err = json.Unmarshal(contents, &poolParamsInput)
	if err != nil {
		return nil, err
	}
```

Reply:
Thanks. Modified in [6a4a3a8](https://github.com/medibloc/panacea-core/pull/287/commits/6a4a3a8d4e4f36ac9fb386eac6b09f47cf48a81b)